src/System Application/App/Azure Key Vault/src/AzureKeyVaultImpl.Codeunit.al

```diff
@@ -30,18 +30,25 @@ codeunit 2202 "Azure Key Vault Impl."
         IsKeyVaultClientInitialized: Boolean;
         AzureKeyVaultTxt: Label 'Azure Key Vault', Locked = true;
         CertificateInfoTxt: Label 'Successfully constructed certificate from secret %1. Certificate thumbprint %2', Locked = true;
+        MissingSecretErr: Label '%1 is either missing or empty', Comment = '%1 = Secret Name.';
```

Comment:
"The secret %1 is either missing or empty."? To make it a bit more readable. Especially in cases where the secret name may be confused for something else being missing.

Reply:
Fair